niceplots/utils.py

```diff
@@ -339,8 +339,8 @@ def horiz_bar(labels, times, header, nd=1, size=[5, 0.5], color=None):
     -------
     fig: matplotlib Figure
         Figure created
-    ax: matplotlib Axes
-        Axes on which data is plotted
+    ax: array of matplotlib Axes
```

Comment:
Can you change the name of this variable in the docstring from `ax` to something to indicate it's more than one axis (like `axs`, `axes`, or `axarr`)